core/src/main/java/hudson/model/UsageStatistics.java

```diff
@@ -95,9 +96,11 @@ public UsageStatistics(String keyImage) {
      * Returns true if it's time for us to check for new version.
      */
     public boolean isDue() {
-        // user opted out. no data collection.
-        if(!Jenkins.getInstance().isUsageStatisticsCollected() || DISABLED)     return false;
-        
+        final Jenkins j = Jenkins.getInstance();
+        // user opted out or Jenkins not fully initialized. no data collection.
+        if(j == null || j.isUsageStatisticsCollected() || DISABLED || COMPLETED.compareTo(j.getInitLevel()) > 0)
```

Comment:
Would be nice if you could update this commit (rebase/force push) to have curly braces and a space around the parentheses. Braces are a bug prevention strategy after all. Otherwise :+1: thanks for adapting to my review comment.